src/modules/campaign-monitor/campaign-monitor.types.ts

```diff
@@ -47,3 +48,14 @@ export interface CampaignMonitorShareFolderInfo {
 export interface CampaignMonitorNewsletterPreferences {
 	newsletter: boolean;
 }
+
+export interface UserInfo {
```

Comment:
REQ: rename to **CampaignMonitorUserInfo**

Tip: Always use module name first and then custom name. We do this because this would not interfere with any other modules that would have similar interface names